unit/util/sharing_node.cpp

```diff
@@ -24,10 +24,10 @@ void sharing_node_internals_test()
     {
       leaft leaf(1, 2);
 
-      auto p = leaf.data.get();
+      /*auto p = leaf.data.get();
       leaf.write();
 
-      REQUIRE(leaf.data.get() == p);
+      REQUIRE(leaf.data.get() == p);*/
```

Comment:
What to make of this? In fact the coding guidelines forbid the use of `/* ... */`. But also if you were to use `#if 0` instead I'd have no idea what is going on here. Needs comments.

Reply:
Ah yes this code is obsolete now. I've removed it.